pom.xml

```diff
@@ -142,6 +142,11 @@ under the License.
             <artifactId>maven-resolver-util</artifactId>
             <version>${resolverVersion}</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.maven.resolver</groupId>
+            <artifactId>maven-resolver-transport-http</artifactId>
+            <version>${resolverVersion}</version>
```

Comment:
This one is in maven 3.9.x and 4.0.x... so maybe "provided"? Or build cache is meant to work with older versions as well?

Reply:
No, build cache requires maven >= 3.9.x, so I'll change it to provided.